From e01cab6703d4b836e6b56e65d14c76ca5c4e66a3 Mon Sep 17 00:00:00 2001
From: Zhang Xiao <xiao.zhang@windriver.com>
Date: Wed, 29 Jun 2016 10:59:40 +0800
Subject: [PATCH] tty/hvc/hvc_console: bug fix for incompatible pointer type

Function pointer poll_init of structure tty_operations add a
parameter but function "hvc_poll_init" not changed to align with
it thus caused this issue. Fix it.

Signed-off-by: Zhang Xiao <xiao.zhang@windriver.com>
---
 drivers/tty/hvc/hvc_console.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/tty/hvc/hvc_console.c b/drivers/tty/hvc/hvc_console.c
index 4fcec1d..568b372 100644
--- a/drivers/tty/hvc/hvc_console.c
+++ b/drivers/tty/hvc/hvc_console.c
@@ -799,7 +799,8 @@ static int hvc_tiocmset(struct tty_struct *tty,
 }
 
 #ifdef CONFIG_CONSOLE_POLL
-static int hvc_poll_init(struct tty_driver *driver, int line, char *options)
+static int hvc_poll_init(struct tty_driver *driver, int line,
+			 char *options, void *rx_callback)
 {
 	return 0;
 }
-- 
1.7.5.4

